controllers/topics-controller.js

```diff
@@ -0,0 +1,16 @@
+const app = require('../app');
+const { fetchTopics } = require('../models/model.js');
+// const { handleServerErrors } = require('./error-handling-controllers')
+
```

Comment:
be careful adding lots of commented out code, it can be quite confusing, its much easier to just remove it and add it again later if you need it than it is to leave trailing comments everywhere :)